ui/ui-components/components/session/SignIn.tsx

```diff
@@ -20,7 +20,10 @@ export default function SignInForm(props) {
     successHandler: SuccessHandler;
     sessionHandler: SessionHandler;
   } = props;
-  const { execApi } = UseApi(props, errorHandler);
+  const { execApi } = useMemo(
+    () => UseApi(props, errorHandler),
+    [errorHandler, props]
+  );
```

Comment:
Since execApi isn't a real hook, it needed to be memoized when used in hooks below.